packages/webview_flutter/webview_flutter_web/README.md

```diff
@@ -22,6 +24,21 @@ yet, so it currently requires extra setup to use:
 Once the step above is complete, the APIs from `webview_flutter` listed
 above can be used as normal on web.
 
+### JavaScript channels
+
+Contrary to the Android and iOS implementations, the `addJavaScriptChannel`
+method does not create a named channel; the channel can only be accessed using
+`window.parent`. In order to send a cross-platform message over a channel named
+`flutterApp`, the following construction can be used in the web application:
+
+```ts
+if (window.flutterApp) {
```

Comment:
What is `window.flutterApp`? Who's setting this?

Reply:
[webview_flutter](https://pub.dev/packages/webview_flutter) is setting this on iOS and Android. If it's not available we assume we are on web and use the alternative.